package.json

```diff
@@ -29,11 +29,16 @@
       "no-shadow": [
         "error",
         {
-          "allow": ["resolve"]
+          "allow": [
+            "resolve"
+          ]
         }
       ],
       "no-use-before-define": "warn",
       "import/no-unresolved": "warn"
     }
+  },
+  "dependencies": {
+    "underscore": "^1.13.6"
```

Comment:
Adding this does not seem to be related to the issue.

Reply:
@StorytellerCZ I've fixed that. Please check once.

Thanks